client_src/js/Tour.mjs

```diff
@@ -117,39 +120,56 @@ class Tour {
 			},
 			buttons: this.get_buttons()
 		});
-		
-		
+
+
 		this.tour.addStep("complete", {
 			text: "Tour complete!\nIf you need any additional assistance, let us know :-)",
-			buttons: [{ text: "Done", action: this.tour.next }]
+			buttons: this.get_buttons(true, false, false)
 		});
 	}
-	
+
 	run_once() {
 		if(window.localStorage.getItem("completed_tour") === null)
 			this.run();
-		
-		window.localStorage.setItem("completed_tour", (new Date()).toISOString());
+
+		//window.localStorage.setItem("completed_tour", (new Date()).toISOString()); -- removed due to an accidental refresh possiblity
 	}
-	
+
 	run() {
 		if(typeof this.tour == "undefined")
 			this.create_tour();
 		this.tour.start();
 	}
-	
-	get_buttons(no_continue = false, no_prev = false) {
+
+	completed_tour()
```

Comment:
Nice refactoring here! This definitely keeps with the existing programming style present.